draft-ietf-quic-http.md

```diff
@@ -246,6 +246,44 @@ DISCUSS:
 do we need to retain compatibility with HTTP/2's HPACK? 
 
 
+### The CONNECT Method
+
+In HTTP/1.x, the pseudo-method CONNECT ({{!RFC7231}}, Section 4.3.6) is used to 
+convert an HTTP connection into a tunnel to a remote host. CONNECT is primarily 
+used with HTTP proxies to establish a TLS session with an origin server for the 
+purposes of interacting with "https" resources. In HTTP/2, the CONNECT method is 
+used to establish a tunnel over a single HTTP/2 stream to a remote host for 
+similar purposes. 
```

Comment:
This was intentional:  "HTTP/1.1 does this; HTTP/2 does this. <para>  Here's how it works in HTTP/QUIC:"  If that's not clear, any suggestions on how to improve it?